kubeflow/pull-upstream.sh

```diff
@@ -134,18 +136,11 @@ kpt pkg get "${KUBEFLOW_MANIFESTS_REPO}/common/user-namespace/@${KUBEFLOW_MANIFE
 rm common/user-namespace/upstream/Kptfile
 
 # contrib/ related manifest
-if [ -d contrib/application/upstream/ ]; then
-    rm -rf contrib/application/upstream/
-fi
-mkdir -p contrib/application
-kpt pkg get "${KUBEFLOW_MANIFESTS_REPO}/contrib/application/@${KUBEFLOW_MANIFESTS_VERSION}" contrib/application/upstream/
-rm contrib/application/upstream/Kptfile
-
 if [ -d contrib/metacontroller/upstream/ ]; then
     rm -rf contrib/metacontroller/upstream/
 fi
 mkdir -p contrib/metacontroller
-kpt pkg get "${KUBEFLOW_MANIFESTS_REPO}/contrib/metacontroller/@${KUBEFLOW_MANIFESTS_VERSION}" contrib/metacontroller/upstream/
+kpt pkg get "${KUBEFLOW_PIPELINES_REPO}/manifests/kustomize/third-party/metacontroller/@${KUBEFLOW_PIPELINES_VERSION}" contrib/metacontroller/upstream/
```

Comment:
NIT: You don't need to take action right now. We can apply metacontroller inside the KFP folder `apps/pipelines` in the future, then you don't need to pull them here.

Reply:
That's a good idea. Thanks. Although current design gives ability for user to choose KFP version in kubeflow directory without going inside kubeflow/apps/pipelines. Let's keep this idea for the next patch release.